tests/test_job_management.py

```diff
@@ -160,7 +168,13 @@ def test_email_callback_after_run_analysis(dispatcher_live_fixture, dispatcher_l
         assert 'email_status' not in jdata
     else:
         assert jdata['exit_status']['email_status'] == 'email sent'
-        smtp_server_log = f'local_smtp_log/{dispatcher_local_mail_server.id}_local_smtp_output.json'
+
+        # check the email in the email folders, and that the first one was produced
+        assert os.path.exists(email_history_folder_path)
+        assert os.path.exists(email_history_folder_path)
+        list_email_files = glob.glob(email_history_folder_path + '/email_0_submitted_*.email')
+        assert len(list_email_files) == 1
```

Comment:
can we simulate the case when one mail was discarded?

Reply:
will do in the next commit